src/common/bip32_check.h

```diff
@@ -2,10 +2,23 @@
 
 #include <stdbool.h>  // bool
 
+#include "../types.h"  // MAX_BIP32_PATH
```

Comment:
No need to use "../"

Reply:
in this specific case, it's mandatory to be able to build (`types.h` vs `../types.h` --> 2 different files I guess).

Reply:
In such case, I suggest to rename the files to avoid more errors...

Reply:
agree but there are several `types.h` in the app and it will require to change many files. It is a bit out of scope of this PR dedicated to the swap --> will have to be handle in another PR.